R/sign_in_components.R

```diff
@@ -26,7 +26,7 @@ sign_in_js <- function(ns) {
       firebase_dependencies(),
       firebase_init(firebase_config),
       tags$script(src = "polish/js/auth_firebase.js?version=4"),
```

Comment:
bump the version to 5

Reply:
6e55a79f72c8e8e49ab74c2a3da1297c7f450763